nautobot/dcim/templates/dcim/map_contact_or_team.html

```diff
@@ -0,0 +1,121 @@
+{% extends 'generic/object_create.html' %}
+{% load form_helpers %}
+{% load helpers %}
+
+{% block form %}
+    <div id="assign" class="tabcontent">
+        <div class="panel panel-default">
+            <div class="panel-heading"><strong>Action</strong></div>
+            <div class="panel-body">
+                {% render_field form.action %}
+            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading"><strong>Location Data</strong></div>
+            <div class="panel-body">
+                {% render_field form.location %}
+                {% render_field form.name %}
+                {% render_field form.phone %}
+                {% render_field form.email %}
+                {% render_field form.address %}
```

Comment:
I'd be inclined to vote against even having an `address` on this form, as it feels like it opens the door to bug reports saying "the address field isn't pre-populated from the location address"?